appinventor/lib/blockly/src/core/blockly.js

```diff
@@ -341,8 +348,7 @@ Blockly.onMouseDown_ = function(e) {
   if (e.target == Blockly.svg && Blockly.isRightButton(e)) {
     // Right-click.
     Blockly.showContextMenu_(e);
-  } else if ((Blockly.readOnly || isTargetSvg) &&
-             Blockly.mainWorkspace.scrollbar) {
+  } else if ((Blockly.readOnly || isTargetSvg) && Blockly.mainWorkspace.scrollbar) {
```

Comment:
Did the code here change, or is this just a reformatting (which if it is, should be undone, as it makes merging new versions of blockly that much harder).